packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -551,6 +551,71 @@ class NavigatorObserver {
 /// because it doesn't depend on its animation parameters (elided with `_`
 /// and `__` in this example). The transition is built on every frame
 /// for its duration.
+///
+/// ### Embedding Navigators
+///
+/// An app can use more than one Navigator. Embedding one Navigator below
+/// another Navigator can result in an "inner journey" such as user
+/// registration, store checkout, or other independent journeys that represent
```

Comment:
I think using an inner navigator to implement registration would be right but isn't necessarily biconditionally right. It could be sensible to do it in a single navigation stack as well (with some usage of popUntil if needed). Perhaps use an example where there are parallel navigation stacks or one navigation stack is only in part of the outer screen etc.

Reply:
Would a tabbed layout with per tab history (like YouTube) be a reasonable example?  Also, do you want me to get rid of the mention of "registration" and "store checkout" or do you just want one more added that has parallel navigation?

Reply:
I'd suggest replacing, since scenarios like parallel navigation stacks and partial screen navigation are only possible using multiple navigators, so it's a bit more clear. 

Per tab history / parallel navigation as an example SG. We do it too in https://github.com/flutter/flutter/blob/9fe53b0d2bb091535c16689cab705c95eac1c53d/packages/flutter/lib/src/cupertino/tab_scaffold.dart

Reply:
@xster is there a canonical example of partial screen navigation?  That is probably the best example that could be provided.  Multiple navigators aren't strictly needed for something like registration, as you pointed out.  But also, parallel navigation for tabs aren't actually nested Navigators - they're just side by side.